src/pods/export/export-table.pod.tsx

```diff
@@ -26,17 +26,25 @@ export const ExportTablePod: React.FC<Props> = props => {
 
   return (
     <>
-      <div className={classes.inputContent}>
+      <fieldset
+        className={classes.inputContent}
+        aria-label="Select an export format"
+        aria-required="true"
+      >
         <div className={classes.radioButton}>
           <input
             className={classes.radioButtonInput}
             type="radio"
             name="export type"
-            id="radio2"
-            aria-label="SVG"
+            id="radio1"
```

Comment:
Because we have a label, we don't need an aria. Delete the arias, and change the id for this radio input with something like, id="radio-svg". Do the same for the other radio buttons.